flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/connectors/hive/read/HivePartitionFetcherContextBase.java

```diff
@@ -202,6 +231,7 @@ public String getComparator() {
 
     @Override
     public void close() throws Exception {
+        partValuesToCreateTime.clear();
```

Comment:
Please check `null` before `clear` in case the `open()` function call failed

Reply:
Nice catch~